console/src/channel/ontology.tsx

```diff
@@ -185,7 +183,7 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
         void Group.fromSelection(props);
         break;
       case "link": {
-        const toCopy = `synnax://cluster/${clusterKey}/channel/${resources[0].id.key}`;
+        const toCopy = `synnax://cluster/${client.key}/channel/${resources[0].id.key}`;
```

Comment:
This won't work. The client key is not equal to the cluster key. Use the cluster key instead.